compute_tools/src/compute.rs

```diff
@@ -161,6 +161,9 @@ pub struct ComputeState {
     pub lfc_prewarm_state: LfcPrewarmState,
     pub lfc_offload_state: LfcOffloadState,
 
+    /// WAL flush LSN that is set after syncing safekeepers after terminating Postgres
+    pub terminate_flush_lsn: Lsn,
```

Comment:
Shouldn't this be Option<Lsn>, given that it's only set during shutdown?

Reply:
Lsn isn't discriminated against Lsn::INVALID so Option<> would increase its size, not sure it's the best option

Reply:
On the other hand, making it a discriminant (or Lsn:::MAX) isn't an option as well, so I'll use your approach.